app/bundles/CoreBundle/Tests/Unit/DependencyInjection/Builder/Metadata/ConfigMetadataTest.php

```diff
@@ -16,8 +16,18 @@ class ConfigMetadataTest extends TestCase
 
     protected function setUp(): void
     {
+        $fieldHelperReflection = new \ReflectionClass(BundleMetadata::class);
+
+        $methods = array_map(static function (\ReflectionMethod $m): string {
+            return $m->name;
+        }, $fieldHelperReflection->getMethods());
```

Comment:
```suggestion
        $methods = array_map(static fn (\ReflectionMethod $m): string => $m->name, $fieldHelperReflection->getMethods());
```
I kinda enjoy the arrow functions. But this is just a nitpick.

Reply:
As I can see that this diff of method names is repeated multiple times in this PR, could you DRY it to some static helper perhaps?

Reply:
Refactored the code, so no reflection is needed.